config.go

```diff
@@ -25,8 +25,9 @@ const (
 	SettingHttpsCertificate = SettingHttps + ".certificate"
 	SettingHttpsKey         = SettingHttps + ".key"
 
-	SettingListen        = "listen"
-	SettingListenDefault = ":8080"
+	SettingListen            = "listen"
+	SettingListenDefault     = ":8080"
+	SettingUsingMultiTenancy = "using_multitenancy"
```

Comment:
Minor thing but we use `have_multitenant` already at he gateway and gui, it would be good to be consistent with naming.

Reply:
From other hand, do we really need that setting? Shouldn't we be able to determine this from request context like we do in other places ?

Reply:
There seems to be some instances where we have multi tenancy enabled, but the token is missing from the context. @mchalski can you fill in the details?

Reply:
hmm, ok, there's two things here.

Maciek is right that we could get rid of this altogether, and just deduce MT / ST from the tenant token. As it stands, the setting just serves to explicitly state our intent and as a constraint for token token verification. so yeah, I missed the larger context here.

@GregorioDiStefano what I meant was - if we *do* have this setting, then we should enforce that the token comes in with a tenant id. if we get rid of it - this is irrelevant, and we go with what we get.

Reply:
...so, my opinion - let's get rid of it if it's not too much effort. sorry @GregorioDiStefano I missed it, I know we're running up to ~30 request changes already...